### DISCUSSION TITLE
Live joint positions

### DISCUSSION THREAD

**Discussion by xPhilippeRoux** [HALRobotics/Beta#97] [Q&A] [answered]
Hi,

Is there a way to get the live value of the joint positions of each axis of a mechanism during toolpath simulation (at a specific target or even in between 2 of them) ? 

Thanks.

**Comment by sebandraos** [comment, marked as answer]
The same trick as for the Parts works here - using Disassemble on your Robot and passing Joints into a num. You'll need to reconnect the Mehcanism to Disassemble to update the values (or use a Timer/Trigger on Disassemble) but for now this is the only option. At some point we will have the ability to Monitor a simulation and extract data but currently the performance penalty of refreshing the Grasshopper components during the simulation is way too big to make it a viable option.
![image](https://user-images.githubusercontent.com/6084391/115034085-05ea8900-9ec3-11eb-92e0-a78fd17d5468.png)
